modules/olm-metrics.adoc

```diff
@@ -30,7 +31,7 @@ state (value `1`) or not (value `0`). Includes the `name`, `namespace`, and
 `version` labels.
 
 |`csv_upgrade_count`
-|Monotonic count of CatalogSources.
+|Monotonic count of CSV upgrades.
```

Comment:
Does someone know what a CSV is at this point?

Reply:
With the new location of this module, I should definitely spell out the first instance.